Formula/scipy.rb

```diff
@@ -46,7 +48,7 @@ def install
     version = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
     ENV["PYTHONPATH"] = Formula["numpy"].opt_lib/"python#{version}/site-packages"
     ENV.prepend_create_path "PYTHONPATH", lib/"python#{version}/site-packages"
-    system Formula["python@3.9"].opt_bin/"python3", "setup.py", "build", "--fcompiler=gnu95"
+    system Formula["python@3.9"].opt_bin/"python3", "setup.py", "build", "--fcompiler=gfortran"
```

Comment:
This depends on `numpy`, though, which also specifies `--fcompiler=gnu95`. Doesn't that need to be fixed first?

Reply:
Yes and oddly I see that on Linux this was not changed to `gfortran`, while it was for `scipy`.  I'm not sure why this isn't consistent, but I believe `gfortran` is the better choice for both.  I will change this in `numpy` first.